downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Actions Cheat Sheet
+byline: GitHub Actions give you the flexibility to build automated software development lifecycle workflows. You can write individual tasks, called actions, and combine them to create  custom workflows in your repository. GitHub Actions are automated processes allowing you to build, test, package, release, or deploy any code project on GitHub, but you can also use them to automate any step of your workflow&#58; merging pull requests, assigning labels, triaging issues to name a few.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Workflow Syntax
+Workflow files use YAML syntax, and must have either a .yml or .yaml file extension. You must store workflow files in the `.github/workflows/` directory of your repository.
+
+```yaml
+name: Node CI
+on: push
+  branches:    
+    - 'releases/*'
+jobs:
+  my_build:
+    runs-on: ubuntu-latest
+    needs: my-other-job
+    steps:
+    - uses: actions/checkout@master
+    - name: Say something
+      run: echo "A little bit more action"
+```
+
+### Workflow `name`
+The name of your workflow. GitHub displays the names of your workflows on your repository's actions page.
+
+### `on` Event
+The name of the GitHub event that triggers the workflow. You can provide a single event string, an array of events, or an event configuration map that restricts the execution of a workflow:
+- When using the `push` and `pull_request` events, `branches` and `tags` allow to select or exclude references the workflow will run on, while `paths` specifies which files must have been modified in order to run the workflow.  
+- The `types` keyword enables you to narrow down activity that causes the workflow to run: `created`, `edited`, `deleted`...
+A workflow trigger can also be scheduled:  
+```yaml
+on:
+  schedule:
+  - cron: '*/15 * * * *'
+```
+
+### `jobs` Collection
+A workflow run is made up of one or more jobs identified by a `job_id`. Jobs  run in parallel by default. Each job runs in a fresh instance of the virtual environment specified by `runs-on`.
+
+#### `job_id`
+The name of the job displayed on GitHub. For example `my_build` or `build`. This string is unique with the `jobs` object.
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colOne | markdownify }}
+</div>
+
+{% capture colTwo %}
+
+#### `needs`
+Identifies any job that must complete successfully before this job will run. It can be a string or array of strings. If a job fails, all jobs that need it are skipped unless the jobs use a conditional statement that causes the job to continue.
+
+#### `runs-on`
+Inside of the `jobs` collection, this is the type of virtual host machine to run the job on.
```

Comment:
```suggestion
The type of virtual host machine to run the job on.  Can be either a GitHub or self-hosted runner.  Jobs can also run in user-specified containers (see: `container`)
```

Reply:
I was going to edit the `inside of the job` but I see @bryancross already proposed a change so :+1: